eagleproject/core/blockchain/harvest/transactions.py

```diff
@@ -162,7 +162,7 @@ def ensure_transaction_and_downstream(tx_hash):
         "debug_data": debug,
         "internal_transactions": internal_transactions,
         "from_address": receipt["from"],
-        "to_address": receipt["to"],
+        "to_address": receipt.get("to"),
```

Comment:
Curious, any reason why this line was changed but not the one above?

Reply:
to address might be missing when there's a contract deployment event, but from address will never be null.

Reply:
Got it